### PR TITLE
Obsolete Frame for NET9 in favor of Border

### PR DESCRIPTION
**Pull request by PureWeen**
### Description of Change

We're at the point with `Border` where we're seeing the majority of `Frame` issues users are having being resolved by using `Border`. For .NET9 we're going to mark `Frame` as obsolete so that we can fully remove `Frame` in .NET 10

### TODO
Add documentation for migrating from Frame to Border

### CODE REVIEW

File: src/Controls/src/Core/Compatibility/Handlers/Android/FrameRenderer.cs [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 namespace Microsoft.Maui.Controls.Handlers.Compatibility
 {
+	[Obsolete("Frame is obsolete as of .NET 9. Please use Border instead.")]
```
Comment (pictos) [inline src/Controls/src/Core/Compatibility/Handlers/Android/FrameRenderer.cs:17]: Would be a good idea to hide it from users with `[EditorBrowsable(EditorBrowsableState.Never)]`?

File: src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue15330.cs [hunk 2]
```diff
@@ -16,6 +16,7 @@ public Issue15330(TestDevice device)
 		[Test]
 		[Category(UITestCategories.Layout)]
 		[FailsOnMac("VerifyScreenshot method not implemented on macOS")]
+		[FailsOnIOS("VerifyScreenshot method not implemented on macOS")]
```
Comment (jfversluis) [inline src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue15330.cs:19]:
```suggestion
		[FailsOnIOS("VerifyScreenshot method not implemented on iOS")]
```

File: src/Controls/src/Core/Compatibility/Handlers/Android/FrameRenderer.cs (hunk 1 above)
Comment (jfversluis) [reply src/Controls/src/Core/Compatibility/Handlers/Android/FrameRenderer.cs:17]: Not against it, but I also don't think it adds a ton of value. The obsolete should show up as a warning and that should be enough. Maybe if anything people will even be confused if they suddenly can't find Frame anymore.

**Review by jfversluis — DISMISSED** [review]

**Comment by PureWeen** [comment]
Device test failures are unrelated and will be fixed by
https://github.com/dotnet/maui/pull/23335

File: src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue15330.cs (hunk 2 above)
Comment (PureWeen) [reply src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue15330.cs]: Updated

File: src/Controls/src/Core/Compatibility/Handlers/Android/FrameRenderer.cs (hunk 1 above)
Comment (PureWeen) [reply src/Controls/src/Core/Compatibility/Handlers/Android/FrameRenderer.cs:17]: Yea, and we're generally just trying to get away from EditorBrowsable

**Comment by PureWeen** [comment]
- Failing iOS tests aren't related and are currently being caused by a flakey lockup

**Comment by jfversluis** [comment]
Huh, guess that doesn't ping @davidbritch anymore... Well, David, maybe something for some doc here!

**Pull request merged**